### PR TITLE
OrganizationView unauthorized fix

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
* Removed `authentication` parameter from `/api/$v1/organizations/{organizationName}/users` endpoint

### CODE REVIEW

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt [hunk 1]
```diff
@@ -74,7 +72,7 @@ class LnkUserOrganizationController(
     @ApiResponse(responseCode = "404", description = "Contest with such name was not found.")
     fun getAllUsersByOrganizationName(
         @PathVariable organizationName: String,
-        authentication: Authentication,
+        authentication: Authentication?,
```
Comment (nulls) [inline save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt:75]: do we need this property in this method?

Comment (sanyavertolet) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt:75]: Actually no, we don't. Will remove it

**Review by nulls — APPROVED** [review]

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt [hunk 2]
```diff
@@ -74,7 +72,6 @@ class LnkUserOrganizationController(
     @ApiResponse(responseCode = "404", description = "Contest with such name was not found.")
     fun getAllUsersByOrganizationName(
         @PathVariable organizationName: String,
-        authentication: Authentication,
```
Comment (Cheshiriks) [inline save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt:77]: Won't an error be thrown when a person enters not authorized?

**Review by Cheshiriks — CHANGES_REQUESTED** [review]

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt:77]: already fixed :)

**Review by Cheshiriks — APPROVED** [review]

**Pull request merged**